### PR TITLE
169 cleanup database classes and query structure

### PR DESCRIPTION
**Pull request by mgdenno**
@samlamont can you take a quick look at this?  Still not great but an improvement I think.

### CODE REVIEW

**Comment by samlamont** [comment]
sure, taking a look now

File: src/teehr/classes/duckdb_base.py [hunk 1]
```diff
@@ -53,20 +54,16 @@ def query(
...
-        if format == "df":
-            return self.con.sql(query).df()
-        elif format == "raw":
-            return self.con.sql(query).show()
-        # elif format == "relation":
-        #     return self.con.sql(query)
-        self.con.sql(query)
-        return None
+        with duckdb.connect() as con:
+            resp = con.cursor().sql(query)
```
Comment (samlamont) [inline src/teehr/classes/duckdb_base.py:61]: Just curious about the use of the cursor here, did you find it was necessary to use to solve a specific problem?

**Review by samlamont — APPROVED** [review]
Looks good to me, just had the question about the cursor object

File: src/teehr/classes/duckdb_base.py (hunk 1 above)
Comment (mgdenno) [reply src/teehr/classes/duckdb_base.py:61]: ah, no good catch, tried it when i was fixing the failing tests and is slipped through when i was cleaning up.

**Pull request merged**